spec/system/comments/user_views_article_comments_spec.rb

```diff
@@ -32,7 +32,7 @@
   end
 
   context "when root is specified" do
-    before { visit "#{article.path}/comments/#{comment.id.to_s(26)}" }
+    before { visit "#{article.path}/comments/#{comment.id.to_s(26)}" } # rubocop:disable Rails/ToSWithArgument
```

Comment:
Rubocop complaining about `to_s(Integer)` ([see this other case](https://github.com/forem/forem/pull/18510/files#r981820372))